src/test/java/me/itzg/helpers/modrinth/ModrinthCommandTest.java

```diff
@@ -378,6 +380,37 @@ void handlesDatapacksLatestVersion(@TempDir Path tempDir) {
             .hasContent("content of zip");
     }
 
+    @Test
+    void usingListingFile(@TempDir Path tempDir) throws Exception {
+        setupStubs();
+
+        final Path listingFile = Files.write(tempDir.resolve("listing.txt"),
+            Arrays.asList(
+                "fabric-api:vNBWcMLP",
+                "# This is a comment",
+                "cloth-config:qA00xo1O",
+                "",
+                "# Another comment"
+            )
+        );
+
+        final int exitCode = new CommandLine(
+            new ModrinthCommand()
+        )
+            .execute(
+                "--api-base-url", wm.getRuntimeInfo().getHttpBaseUrl(),
+                "--output-directory", tempDir.toString(),
+                "--game-version", "1.21.5",
+                "--loader", "fabric",
+                "--projects", "@" + listingFile
+            );
+
+        assertThat(exitCode).isEqualTo(ExitCode.OK);
```

Comment:
The builds in GitHub don't seem to reveal the actual failure, so I'll see if it fails for me locally too

https://github.com/itzg/mc-image-helper/actions/runs/15746318998/job/44383891924 Add Modrinth project files support · itzg/mc-image-helper@d844d66